### PR TITLE
Updated Italian Translation

### PR DESCRIPTION
**Pull request by raffas**
Improved Italian translation

### CODE REVIEW

File: superset/translations/it/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -120,15 +121,15 @@ msgstr "10 minuti"
 #: superset/utils.py:499
 #, python-format
 msgid "[Superset] Access to the datasource %(name)s was granted"
-msgstr ""
+msgstr "[Siperset] Accesso al datasource $(name) concesso"
```
Comment (timifasubaa) [inline superset/translations/it/LC_MESSAGES/messages.po]: Should this be `Superset` or is my Italian horrible?

File: superset/translations/it/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -1171,68 +1172,68 @@ msgstr ""
...
 
 #: superset/assets/javascripts/explore/components/controls/VizTypeControl.jsx:110
 msgid "Select a visualization type"
-msgstr ""
+msgstr "Seleziona un tipo di visualizzazione"
 
 #: superset/assets/javascripts/explore/reducers/chartReducer.js:32
 msgid "Updating chart was stopped"
-msgstr ""
+msgstr "L'aggiornamento del grafico &egrave; stato fermato"
```
Comment (xrmx) [inline superset/translations/it/LC_MESSAGES/messages.po]: letters with accents

File: superset/translations/it/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -1244,19 +1245,19 @@ msgstr ""
 
 #: superset/assets/javascripts/explore/reducers/chartReducer.js:56
 msgid "Network error."
-msgstr ""
+msgstr "Errore di rete"
```
Comment (xrmx) [inline superset/translations/it/LC_MESSAGES/messages.po]: misisng dot

File: superset/translations/it/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -1244,19 +1245,19 @@ msgstr ""
...
 
 #: superset/assets/javascripts/explore/stores/controls.jsx:50
 msgid "Group by"
-msgstr ""
+msgstr "Raggruppa per"
 
 #: superset/assets/javascripts/explore/stores/controls.jsx:53
 msgid "One or many controls to group by"
-msgstr ""
+msgstr "Uno o pi&ugrave; controlli per 'Raggruppa per'"
```
Comment (xrmx) [inline superset/translations/it/LC_MESSAGES/messages.po]: letter with accents please

File: superset/translations/it/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -1271,20 +1272,20 @@ msgstr "Tipo di Visualizzazione"
...
 #: superset/assets/javascripts/explore/stores/controls.jsx:90
 msgid "Metrics"
-msgstr ""
+msgstr "Metriche"
 
 #: superset/assets/javascripts/explore/stores/controls.jsx:99
 #: superset/assets/javascripts/explore/stores/controls.jsx:116
 msgid "One or many metrics to display"
-msgstr ""
+msgstr "Una o pi&ugrave; metriche da mostrare"
```
Comment (xrmx) [inline superset/translations/it/LC_MESSAGES/messages.po]: same

File: superset/translations/it/LC_MESSAGES/messages.po [hunk 6]
```diff
@@ -2274,16 +2275,16 @@ msgstr ""
...
 #: superset/assets/javascripts/explore/stores/visTypes.js:9
 #: superset/assets/javascripts/explore/stores/visTypes.js:32
 msgid "Time related form attributes"
-msgstr ""
+msgstr "Attributi relativi al tempo"
 
 #: superset/assets/javascripts/explore/stores/visTypes.js:16
 msgid "Datasource & Chart Type"
-msgstr ""
+msgstr "Sorgente dati e Tipo di grafico"
```
Comment (xrmx) [inline superset/translations/it/LC_MESSAGES/messages.po]: Lowercase tipo. In other places you haven't translated datasource, i think we should decide to translate or not and be consistent. But we could revise this later.

File: superset/translations/it/LC_MESSAGES/messages.po [hunk 7]
```diff
@@ -2296,7 +2297,7 @@ msgstr ""
 
 #: superset/assets/javascripts/explore/stores/visTypes.js:66
 msgid "Advanced Analytics"
-msgstr ""
+msgstr "Analitycs avanzato"
```
Comment (xrmx) [inline superset/translations/it/LC_MESSAGES/messages.po]: Analytics avanzate

File: superset/translations/it/LC_MESSAGES/messages.po [hunk 8]
```diff
@@ -2343,48 +2344,48 @@ msgstr ""
...
+msgstr "RAGGRUPPA PER"
 
 #: superset/assets/javascripts/explore/stores/visTypes.js:338
 msgid "Use this section if you want a query that aggregates"
 msgstr ""
 
 #: superset/assets/javascripts/explore/stores/visTypes.js:346
 msgid "NOT GROUPED BY"
-msgstr ""
+msgstr "NON RAGGRUPARE PER"
```
Comment (xrmx) [inline superset/translations/it/LC_MESSAGES/messages.po]: raggrup**p**are

File: superset/translations/it/LC_MESSAGES/messages.po [hunk 9]
```diff
@@ -2464,15 +2465,15 @@ msgstr ""
 
 #: superset/assets/javascripts/explore/stores/visTypes.js:794
 msgid "Source"
-msgstr ""
+msgstr "Sorgente"
 
 #: superset/assets/javascripts/explore/stores/visTypes.js:797
 msgid "Choose a source"
 msgstr ""
```
Comment (xrmx) [inline superset/translations/it/LC_MESSAGES/messages.po]: missed translation?

**Review by xrmx — CHANGES_REQUESTED** [review]

**Comment by raffas** [comment]
@xrmx Updated translation with your changes

File: superset/translations/it/LC_MESSAGES/messages.po [hunk 10]
```diff
@@ -391,83 +391,83 @@ msgstr ""
...
 #: superset/assets/javascripts/SqlLab/actions.js:297
 #: superset/assets/javascripts/SqlLab/actions.js:310
 msgid "Error occurred while fetching table metadata"
-msgstr ""
+msgstr "Errore nel recupero dei metadati della tabella"
 
 #: superset/assets/javascripts/SqlLab/actions.js:364
 msgid "shared query"
-msgstr ""
+msgstr "tabella condivisa"
```
Comment (xrmx) [inline superset/translations/it/LC_MESSAGES/messages.po]: query is query

File: superset/translations/it/LC_MESSAGES/messages.po [hunk 11]
```diff
@@ -391,83 +391,83 @@ msgstr ""
...
 #: superset/assets/javascripts/SqlLab/reducers.js:44
 #, python-format
 msgid "Copy of %s"
-msgstr ""
+msgstr "Copia di %s"
 
 #: superset/assets/javascripts/SqlLab/components/CopyQueryTabUrl.jsx:30
 msgid "share query"
-msgstr ""
+msgstr "query condivisa"
```
Comment (xrmx) [inline superset/translations/it/LC_MESSAGES/messages.po]: condividi query

File: superset/translations/it/LC_MESSAGES/messages.po [hunk 12]
```diff
@@ -391,83 +391,83 @@ msgstr ""
...
 
 #: superset/assets/javascripts/SqlLab/components/CopyQueryTabUrl.jsx:33
 msgid "copy URL to clipboard"
-msgstr ""
+msgstr "copia URL in clipoboard"
```
Comment (xrmx) [inline superset/translations/it/LC_MESSAGES/messages.po]: copia URL negli appunti

File: superset/translations/it/LC_MESSAGES/messages.po [hunk 13]
```diff
@@ -2296,7 +2296,7 @@ msgstr ""
 
 #: superset/assets/javascripts/explore/stores/visTypes.js:66
 msgid "Advanced Analytics"
-msgstr ""
+msgstr "Analitycs avanzate"
```
Comment (xrmx) [inline superset/translations/it/LC_MESSAGES/messages.po]: Analytics, you swap y with i :)

**Review by xrmx — CHANGES_REQUESTED** [review]

**Comment by xrmx** [comment]
Thanks a lot i've found a few more typos! With these fixed, looks good to me, thanks a lot!

**Comment by raffas** [comment]
@xrmx Translation updated

**Comment by xrmx** [comment]
@raffas thanks!

**Pull request merged**